import_export_celery/models/importjob.py

```diff
@@ -71,9 +71,7 @@ class ImportJob(models.Model):
     model = models.CharField(
         verbose_name=_("Name of model to import to"),
         max_length=160,
-        choices=[
-            (x, x) for x in getattr(settings, "IMPORT_EXPORT_CELERY_MODELS", {}).keys()
-        ],
+        choices=[],
```

Comment:
This line can be deleted, right?

Reply:
@timthelion Yes it can now that I've had to use a choice field on the form 👍